devel/ci/rawhide-packages

```diff
@@ -10,4 +10,6 @@
     python3-cornice \
     python3-cornice-sphinx \
     python2-webtest \
-    python3-pyramid-fas-openid
+    python3-pyramid-fas-openid \
+    python2-hawkey \
+    python2-libcomps
```

Comment:
I have a feeling the Python 3 versions of these packages are getting pulled in via ```requirements.txt``` instead of via RPM packages. We should probably add those to ```devel/bodhi/devel/ci/rpm-packages```.

Reply:
They are in the base system as they're used by DNF...

Reply:
Ah cool